From: Werner Koch <wk@gnupg.org>
Date: Wed, 5 Jul 2017 10:08:13 +0200
Subject: build: Install yat2m.

* doc/Makefile.am (install-exec-hook): New.
(uninstall-local): Uninstall yat2m.

Signed-off-by: Werner Koch <wk@gnupg.org>
(cherry picked from commit d81b1379edbc6007a89c3b2fff1b51c8dfe2e868)
---
 doc/Makefile.am | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index c914527..d052283 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -72,9 +72,22 @@ errorref.txt.x : errorref.txt
 	sed '/^##/ d' $< >$@
 	echo "# Installed by $(PACKAGE_NAME) $(PACKAGE_VERSION)" >>$@
 
+install-exec-hook:
+if CROSS_COMPILING
+	@echo "not install yat2m while cross-compiling"
+else
+	@echo "installing yat2m on the build system"; \
+         $(MKDIR_P) "$(DESTDIR)$(bindir)"; \
+	 $(INSTALL_PROGRAM_ENV) $(INSTALL_PROGRAM) \
+	         yat2m "$(DESTDIR)$(bindir)/yat2m"
+endif
+
 install-data-local: errorref.txt.x
 	$(mkinstalldirs) $(DESTDIR)$(pkgdatadir)
 	$(INSTALL_DATA) errorref.txt.x $(DESTDIR)$(pkgdatadir)/errorref.txt
 
 uninstall-local:
 	-@rm $(DESTDIR)$(pkgdatadir)/errorref.txt
+if !CROSS_COMPILING
+	-@rm $(DESTDIR)$(bindir)/yat2m
+endif
